stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/exomedia/core/source/MediaSourceProvider.java

```diff
@@ -39,11 +39,11 @@
  */
 @SuppressWarnings("WeakerAccess")
 public class MediaSourceProvider {
-    protected static final String USER_AGENT_FORMAT = "ExoMedia %s (%d) / Android %s / %s";
+    protected static final String USER_AGENT_FORMAT = "ExoMedia %s / Android %s / %s";
 
     @NonNull
     @SuppressLint("DefaultLocale")
-    protected String userAgent = String.format(USER_AGENT_FORMAT, BuildConfig.VERSION_NAME, BuildConfig.VERSION_CODE, Build.VERSION.RELEASE, Build.MODEL);
+    protected String userAgent = String.format(USER_AGENT_FORMAT, BuildConfig.STREAM_CHAT_UI_VERSION, Build.VERSION.RELEASE, Build.MODEL);
```

Comment:
As I understand, the user agent here is an arbitrary string, and the version name should be enough to identify the library version in logs and such.